src/kudu/consensus/consensus_peers.cc

```diff
@@ -92,6 +92,10 @@ DEFINE_bool(enable_tablet_copy, true,
             "replica. For testing purposes only.");
 TAG_FLAG(enable_tablet_copy, unsafe);
 
+DEFINE_int32(raft_proxy_max_hops, 16,
```

Comment:
nit: is 16 too high?

Reply:
16 is seemingly high but not huge (for comparison, IP recommends a default TTL of 64). Putting a kind-of high number means -- hopefully -- no one will ever have to use this parameter here. The trade-off is that it also means we can have 16x amplification of traffic while a bad config is in place on the replicaset... so maybe 8 is a better number?